home.html

```diff
@@ -83,15 +83,25 @@ <h1 class="heading2">Category</h1>
         <div class="category-list">
           <div class="shoe-category category-item">
             <img src="/assets/shoes-cover.avif" alt="cover image" />
-            <a href="#" class="font-bold shadow-card category-name">Shoes</a>
+            <a
+              href="/pages/products.html"
+              class="font-bold shadow-card category-name"
+              >Shoes</a
```

Comment:
you can increase line width in prettier extension so formatting will be good ideally it's 80 i'm using 100. Not essential just suggestion